packages/core/src/dialects/abstract/query-interface.d.ts

```diff
@@ -300,25 +300,6 @@ export class AbstractQueryInterface extends AbstractQueryInterfaceTypeScript {
 
   constructor(sequelize: Sequelize, queryGenerator: AbstractQueryGenerator);
 
-  /**
-   * Drops the specified schema (table).
-   *
-   * @param schema The schema to query. Applies only to Postgres.
-   */
-  dropSchema(schema?: string, options?: QueryRawOptions): Promise<void>;
-
-  /**
-   * Drops all tables.
-   */
-  dropAllSchemas(options?: QueryInterfaceDropAllTablesOptions): Promise<void>;
```

Comment:
This should remain now that we're migrating it later

Reply:
Done